modules/installation-gcp-config-yaml.adoc

```diff
@@ -17,6 +17,11 @@ ifeval::["{context}" == "installing-gcp-private"]
 :private:
```

Comment:
network, controlPlaneSubnet and computeSubnet is not required in install-config.yaml. If you specify them, you will get the below error as the network is in host project instead of service project. 
FATAL failed to fetch Master Machines: failed to load asset "Install Config": platform.gcp.network: Invalid value: "aos-qe-network": failed to get network aos-qe-network: googleapi: Error 404: The resource 'projects/openshift-qe/global/networks/aos-qe-network' was not found, notFound